create_dataset/combine_dataset.py

```diff
@@ -0,0 +1,68 @@
+"""
+For our experiments, we parallelize the creation of peak datasets (traces from all channels around
+a detected peak) into N tasks. Each task generates an HDF5 file with two datasets: "properties" and
+"traces". This script combines the datasets from those N HDF5 files into one HDF5 file named 
+"peaks.h5" and adds another dataset, "channel_locations". 
+"""
+import argparse
+import os
+
+import numpy as np
+import h5py
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Combine individual HDF5 files into a single file.")
+    parser.add_argument('recording_id', type=str, help="ID of the recording.")
+    parser.add_argument('num_tasks', type=int, default=1, help="Number of tasks used in processing.")
```

Comment:
Change this so that it reads all `peaks_x.h5` files instead of requiring the user to specify the number of tasks.